.platform/solr_config/solrconfig_extra.xml

```diff
@@ -1,188 +1,190 @@
-    <str name="df">id</str>
-    <str name="spellcheck.dictionary">und</str>
-    <str name="spellcheck">on</str>
-    <str name="spellcheck.onlyMorePopular">false</str>
-    <str name="spellcheck.extendedResults">false</str>
-    <str name="spellcheck.count">1</str>
-    <str name="spellcheck.alternativeTermCount">5</str>
-    <str name="spellcheck.maxResultsForSuggest">5</str>
-    <str name="spellcheck.collate">true</str>
-    <str name="spellcheck.collateExtendedResults">true</str>
-    <str name="spellcheck.maxCollationTries">10</str>
-    <str name="spellcheck.maxCollations">5</str>
-  </lst>
-  <arr name="last-components">
-    <str>spellcheck</str>
-  </arr>
+<lst name="defaults">
+  <str name="df">id</str>
+  <str name="spellcheck.dictionary">und</str>
+  <str name="spellcheck">on</str>
+  <str name="spellcheck.onlyMorePopular">false</str>
+  <str name="spellcheck.extendedResults">false</str>
+  <str name="spellcheck.count">1</str>
+  <str name="spellcheck.alternativeTermCount">5</str>
+  <str name="spellcheck.maxResultsForSuggest">5</str>
+  <str name="spellcheck.collate">true</str>
+  <str name="spellcheck.collateExtendedResults">true</str>
+  <str name="spellcheck.maxCollationTries">10</str>
+  <str name="spellcheck.maxCollations">5</str>
+</lst>
+<arr name="last-components">
+  <str>spellcheck</str>
+</arr>
 </requestHandler>
 
-<!--
-  Suggester
-  7.0.0
--->
+  <!--
+    Suggester
+    7.0.0
+  -->
 <requestHandler name="/suggest" class="solr.SearchHandler" startup="lazy">
-  <lst name="defaults">
-    <str name="suggest">true</str>
-    <str name="suggest.dictionary">und</str>
-    <str name="suggest.dictionary">10</str>
-  </lst>
-  <arr name="components">
-    <str>suggest</str>
-  </arr>
+<lst name="defaults">
+  <str name="suggest">true</str>
+  <str name="suggest.dictionary">und</str>
+  <str name="suggest.dictionary">10</str>
+</lst>
+<arr name="components">
+  <str>suggest</str>
+</arr>
 </requestHandler>
 
-<!--
-  Term Vector
-  7.0.0
--->
+  <!--
+    Term Vector
+    7.0.0
+  -->
 <requestHandler name="/tvrh" class="solr.SearchHandler" startup="lazy">
-  <lst name="defaults">
-    <str name="df">id</str>
-    <str name="tv">true</str>
-  </lst>
-  <arr name="last-components">
-    <str>tvComponent</str>
-  </arr>
+<lst name="defaults">
+  <str name="df">id</str>
+  <str name="tv">true</str>
+</lst>
+<arr name="last-components">
+  <str>tvComponent</str>
+</arr>
 </requestHandler>
 
-<!--
-  Special configs for Elevator
-  7.0.0
--->
-  <searchComponent name="elevator" class="solr.QueryElevationComponent">
-    <str name="queryFieldType">string</str>
-    <str name="config-file">elevate.xml</str>
-  </searchComponent>
-<!--
-  Special configs for Term Vector
-  7.0.0
--->
-  <searchComponent name="tvComponent" class="solr.TermVectorComponent"/>
+  <!--
+    Special configs for Elevator
+    7.0.0
+  -->
+<searchComponent name="elevator" class="solr.QueryElevationComponent">
+<str name="queryFieldType">string</str>
```

Comment:
Can this queryFieldType be changed from string to text_ws?

When set to "string", the elevator component doesn't do any special processing of the query terms before comparing them with elevate.xml entries.

So if I search for "Go ON NI", the elevator component will not elevate the entry for "go on ni" because of the case difference between the search terms and query text specified for the elevate entry.

The text_ws field type does some tokenisation and converts query terms to lower case.  Seems to be present in 7.x schema.xml so should be safe to use it.

```
    <!-- A text field that only splits on whitespace for exact matching of words -->
    <fieldType name="text_ws" class="solr.TextField" omitNorms="true" positionIncrementGap="100">
      <analyzer>
        <tokenizer class="solr.WhitespaceTokenizerFactory"/>
        <filter class="solr.LowerCaseFilterFactory"/>
      </analyzer>
    </fieldType>
```

Reply:
Yes, you're quite right here. I've got this open in front of me today after I had the extra information through from yourself and @David-Corbett yesterday. No hurry to commit this at the moment so I'll evaluate and commit changes, then re-request a review once it looks about right.